tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "skipLibCheck": true,
+    "noStrictGenericChecks": true,
+    "strictFunctionTypes": false
```

Comment:
Eh, why do we need to change these?

Reply:
ah... some config that I forgot to remove when debugging an issue. removed now.